services/src/test/java/io/scalecube/services/GreetingServiceImpl.java

```diff
@@ -9,6 +9,11 @@
 final class GreetingServiceImpl implements GreetingService {
 
   @Override
+  public String toString() {
+    return "GreetingServiceImpl []";
```

Comment:
Why do you need toString for service? Also default implementation of toString will be about the same informative